src/main/java/org/assertj/core/api/AbstractIterableAssert.java

```diff
@@ -2754,6 +2754,51 @@ public SELF allSatisfy(Consumer<? super ELEMENT> requirements) {
     return myself;
   }
 
+  /**
+   * Verifies that the all the given consumers can be satisfied by elements in the iterable with
+   * an element at most satisfies one consumer. No order requirement.
+   * <p>
+   * This assertion is to test whether there are enough elements in the iterable that can satisfy all the consumers.
+   * <p>
+   * Grouping assertions example :
+   * <pre><code class='java'> // second constructor parameter is the light saber color
```

Comment:
Since the examples is not primarily about grouping, maybe just call them "Examples"?
Since we just create a list of Strings, is there a copy/paste error about the second constructor parameter? 

```suggestion
   * Examples:
   * <pre><code class='java'>
```